src/user-profile/store/effects/user.effects.ts

```diff
@@ -7,16 +7,16 @@ import {UserService} from '../../services/user.service';
 import {AuthActionTypes} from '../actions/';
 import {UserInterface} from '../../models/user.model';
 import {HttpErrorResponse} from '@angular/common/http';
-import config from '../../../../api/lib/config';
 import * as usersActions from '../../../users/store/actions/user.actions';
 import { UserRolesUtil } from 'src/users/containers/utils/user-roles-util';
-import * as fromRoot from '../../../app/store';
+import { LoggerService } from 'src/shared/services/logger.service';
```

Comment:
I think we should have a standard way relative route references. `../../` instead of `src/`. The line above this one as  well.

Reply:
Done